docs/pages/router/reference/typed-routes.mdx

```diff
@@ -138,7 +138,7 @@ When typed routes is enabled, Expo CLI will generate a git ignored **expo-env.d.
 
 The `includes` field in your **tsconfig.json** gets updated to include **expo-env.d.ts** and a hidden **.expo** directory. These entries are required and should not be removed from the file.
 
-The generated **expo-env.d.ts** should not be removed or changed at any time. It should not be committed and should be ignored by version control. For example, don't add it inside the **.gitignore**.
+The generated **expo-env.d.ts** should not be removed or changed at any time. It should not be committed and should be ignored by version control. For example, add it inside the **.gitignore**.
```

Comment:
```suggestion
The generated **expo-env.d.ts** should not be removed or changed at any time. It should not be committed and should be ignored by version control.
```

Hey @Randall71, thanks for catching this. Since this step is done automatically by the Expo CLI when `npx expo start` is run for the first time in a project. I understand it's a bit confusing whether to add this file to gitignore or whether it is done automatically. However, as the first paragraph says that this file is automatically added, let's remove this example line.

Also, here is the screenshot of that happening:

![CleanShot 2023-08-28 at 14 48 04](https://github.com/expo/expo/assets/10234615/caba6be2-0c52-422f-897a-a7fd9a61aeff)

This is tested using both quick and manual steps from Expo Router docs.